doc/languages-frameworks/python.section.md

```diff
@@ -70,19 +70,19 @@ temporary shell session with a Python and a *precise* list of packages (plus
 their runtime dependencies), with no other Python packages in the Python
 interpreter's scope.
 
-To create a Python 3.8 session with `numpy` and `toolz` available, run:
+To create a Python 3.9 session with `numpy` and `toolz` available, run:
 
 ```sh
-$ nix-shell -p 'python38.withPackages(ps: with ps; [ numpy toolz ])'
+$ nix-shell -p 'python39.withPackages(ps: with ps; [ numpy toolz ])'
```

Comment:
How about we save ourselves from updating this?

```suggestion
$ nix-shell -p 'python3.withPackages(ps: with ps; [ numpy toolz ])'
```

Reply:
because that does not correspond to the description of the example

Reply:
OFC that would also be changed to only mention python 3. IMO we don't need to update half the documentation when updating the default python version.